contracts/test/forkMainnet/AMMWrapper/Setup.t.sol

```diff
@@ -134,25 +135,31 @@ contract TestAMMWrapper is StrategySharedSetup {
         sig = abi.encodePacked(r, s, v, bytes32(0), uint8(2));
     }
 
+    function _createSpenderPermitFromOrder(AMMLibEIP712.Order memory defaultOrder)
+        internal
+        view
+        returns (SpenderLibEIP712.SpendWithPermit memory takerAssetPermit)
+    {
+        // Declare the 'userAddr sends takerAssetAmount amount of takerAssetAddr to AMM contract'
+        // SpendWithPermit struct from defaultOrder parameter
```

Comment:
nit: I think these two lines of comments could be removed since it doesn't provide more information than code itself.